homeassistant/components/switchbee/config_flow.py

```diff
@@ -33,7 +38,7 @@
 
 STEP_ADVANCED_DATA_SCHEMA = vol.Schema(
     {
-        vol.Required(CONF_SCAN_INTERVAL, default=5): cv.positive_int,
+        vol.Required(CONF_SCAN_INTERVAL, default=SCAN_INTERVAL_SEC): cv.positive_int,
         vol.Required(CONF_EXPOSE_SCENARIOS, default=False): cv.boolean,
         vol.Required(CONF_EXPOSE_GROUP_SWITCHES, default=False): cv.boolean,
     }
```

Comment:
These should not be set via config flow, but via config options https://developers.home-assistant.io/docs/config_entries_options_flow_handler

I highly suggest to focus on making the PR small as possible and try not to add additional functionality during review (unless this is needed to fix something raised in the review). 

Thanks 👍